packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1594,9 +1607,9 @@ async function handler(
       if (booking && booking.id && eventType.seatsPerTimeSlot) {
         const currentAttendee = booking.attendees.find(
           (attendee) =>
-            attendee.email === input.bookingData.responses.email ||
-            (input.bookingData.responses.attendeePhoneNumber &&
-              attendee.phoneNumber === input.bookingData.responses.attendeePhoneNumber)
+            attendee.email === bookingData.responses.email ||
```

Comment:
Using _bookingData_ instead of _input.bookingData_ here because bookingData is parsed by schema and thus has responses always while _input.bookingData_ doesn't have it always.

In a followup PR we are removing input.bookingData usage from everwhere.